rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -76,17 +75,17 @@ endforeach()
 
 if(NOT "${_generated_msg_py_files} " STREQUAL " ")
   list(GET _generated_msg_py_files 0 _msg_file)
-  get_filename_component(_parent_folder "${_msg_file}" DIRECTORY)
+  get_filename_component(_parent_folder_path "${_msg_file}" DIRECTORY)
   list(APPEND _generated_msg_py_files
-    "${_parent_folder}/__init__.py"
+    "${_parent_folder_path}/__init__.py"
```

Comment:
If not necessary please avoid changing variable names like this since it make reading the diff more difficult since the reader needs to separate essential and non-essential changes.

Same a few lines below.